test/gtest/ucp/ucp_test.h

```diff
@@ -7,6 +7,7 @@
 #define UCP_TEST_H_
 
 #include <ucp/api/ucp.h>
+#include <ucp/api/ucpx.h>
```

Comment:
didn't we remove this?

Reply:
I'll fix this and the UCP_FEATURE_EXPERIMENTAL part when the other PR gets merged.

Reply:
np